tensorflow/python/distribute/parameter_server_strategy.py

```diff
@@ -123,6 +123,9 @@ def __init__(self, cluster_resolver=None):
         len(self.extended.parameter_devices))
 
   def experimental_distribute_dataset(self, dataset, options=None):
+    if options and options.replication_mode == distribute_lib.InputReplicationMode.PER_REPLICA:
```

Comment:
Can you also add this error to tpu_strategy.py ?

Reply:
Added.